ebusd-2.1.x/de/vaillant/08.recov.csv

```diff
@@ -0,0 +1,38 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,recov,recoVair 260,,,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B513,,,,IGN:2,,,
+*w,,,,,,B509,,,,,,,
+r,recov,VentIntensityDay,Lueftungsintensitaet Tag auslesen,,,,048a03,,,UCH,,,
+r,recov,VentIntensityNight,Lueftungsintensitaet Nacht auslesen,,,,048b03,,,UCH,,,
+r,recov,VentHeatRecovery,Waermerueckgewinnung auslesen,,,,048c03,,,UCH,0=auto;1=an;2=aus,,
+w,recov,SetVentIntensityDay1,Lueftungsintensitaet Tag 1 setzen,,,,0e8a0301,,,,,,
```

Comment:
this should be replaced with a intensity parameter and combined with the read

Reply:
How could I to this?